### PR TITLE
Show read-only user-attributes as locked

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8840

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/components/QuotaSelect.vue [hunk 1]
```diff
@@ -4,21 +4,17 @@
...
       push-tags
       :clearable="false"
       :options="options"
       :create-option="createOption"
       option-label="displayValue"
-      :label="title"
-      :fix-message-line="fixMessageLine"
-      :error-message="errorMessage"
-      :warning-message="warningMessage"
-      :description-message="descriptionMessage"
```
Comment (JanAckermann) [inline packages/web-pkg/src/components/QuotaSelect.vue:18]: basically I do the same as we do with oc-select and vue-select

File: packages/web-pkg/src/components/QuotaSelect.vue [hunk 2]
```diff
@@ -40,51 +36,13 @@ import { formatFileSize } from 'web-pkg'
...
-      type: String,
-      default: null
-    },
-    /**
-     * A description text which is shown below the select field.
-     */
-    descriptionMessage: {
-      type: String,
-      default: null
     }
```
Comment (JanAckermann) [inline packages/web-pkg/src/components/QuotaSelect.vue:46]: These are props of oc-select, we don't need to define them twice =)

File: packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue [hunk 3]
```diff
@@ -161,11 +168,15 @@ export default defineComponent({
         (g) => !selectedGroups.some((s) => s.id === g.id) && !g.groupTypes?.includes('ReadOnly')
       )
     })
-
     const isLoginInputDisabled = computed(() => currentUser.uuid === (props.user as User).id)
+    const readOnlyUserAttributes = useCapabilityReadOnlyUserAttributes()
+    const isInputFieldReadOnly = (key) => {
+      return readOnlyUserAttributes.value.includes(key)
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue:174]: Quick reminder that we usually prefer `unref()` :)

```suggestion
      return unref(readOnlyUserAttributes).includes(key)
```

**Review by JammingBen — APPROVED** [review]
LGTM

**Pull request merged**